src/.vuepress/.idea/.idea..vuepress.dir/.idea/indexLayout.xml

```diff
@@ -0,0 +1,8 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
What are these .idea folders? Why are they in your PR? If they're not supposed to be here update the project level .gitignore file and fix the PR to remove them.